### DISCUSSION TITLE
Add community files to laravel and illuminate organisations

### DISCUSSION THREAD

**Discussion by lucasmichot** [laravel/framework#40766] [General]
The `laravel` and `illuminate` organisations could come with a global org `readme` files, and some communities files.

See:
- https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file
- https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile

---

Here is an example of what a fancy `README.md` could look like:

![image](https://user-images.githubusercontent.com/513603/152220203-494597d1-02ea-493a-a49c-4b2dfb2afae1.png)

**Comment by chuckrincon** [comment]
It would be great to have this.

Pinging the team to see if it is possible:
@taylorotwell @themsaid @driesvints @jbrooksuk

**Reply by ousid** [reply]
@nunomaduro

**Comment by driesvints** [comment]
I'll check with us internally to see what we can do here.
